mysql-test/main/func_str.result

```diff
@@ -5269,1032 +5269,18 @@ f
 # MDEV-25704 Function random_bytes
 #
 create table t1 as select random_bytes(100);
-select seq from seq_1_to_1024 where not (random_bytes(seq) <=> random_bytes(seq)) and length(random_bytes(seq)) = seq;
-seq
+show create table t1;
+Table	Create Table
+t1	CREATE TABLE `t1` (
+  `random_bytes(100)` longblob DEFAULT NULL
```

Comment:
Thanks. Now one can see the issue. Logically, one can expect the result of `random_bytes(100)` to be `varbinary(100) not null` (or `varchar(100) not null character set binary` which is the same thing)

Reply:
Then why one wouldn't expect the same with any other string function? For example, consider REPEAT output:
```
create table t1 as select repeat('1', 10);
show create table t1;
Table	Create Table
t1	CREATE TABLE `t1` (
  `repeat('1', 10)` varchar(10) DEFAULT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1
```

Reply:
Do you mean NULL/NOT NULL? Yes, I'd think `repeat` should create a NOT NULL column too.

But my main point was to create a `varbinary(100)` column, not `longblob`. Like `repeat` does.